deps/patches/llvm-D88630-clang-cmake.patch

```diff
@@ -0,0 +1,29 @@
+commit 9c682bac219e28afc73e25e7089b553ecd1ca0f6
+Author: Keno Fischer <keno@juliacomputing.com>
+Date:   Wed Sep 30 22:15:40 2020 -0400
+
+    [clang/CMake] Respect LLVM_TOOLS_INSTALL_DIR
+    
+    Otherwise clang installs all of its tools into `bin/` while
+    LLVM installs its tools into (LLVM_TOOLS_INSTALL_DIR).
+    I could swear this used to work (and in fact the julia build system
```

Comment:
Yeah the pre-built binaries assume this as well https://github.com/JuliaPackaging/Yggdrasil/blob/a7802b8c7a03f5d43ba05937960ad93da4efde3c/L/LLVM/common.jl#L142-L146

But apparently we fix this post-build https://github.com/JuliaPackaging/Yggdrasil/blob/a7802b8c7a03f5d43ba05937960ad93da4efde3c/L/LLVM/common.jl#L220-L225